support-frontend/assets/components/checkoutTopUpToggle/checkoutTopUpToggle.tsx

```diff
@@ -0,0 +1,49 @@
+import { SvgTickRound } from '@guardian/source-react-components';
+import { ToggleSwitch } from '@guardian/source-react-components-development-kitchen';
+import type { RegularContributionType } from 'helpers/contributions';
+import {
+	standfirst,
+	title,
+	toggleSwitchOverrides,
+	topUpToggleContainer,
+} from './checkoutTopUpToggleStyles';
+
+const timePeriods = {
+	MONTHLY: 'month',
+	ANNUAL: 'year',
+};
+
+export interface CheckoutTopUpToggleProps {
+	checked: boolean;
+	onChange(): void;
+	contributionType: RegularContributionType;
+	threshold: number;
+}
+
+export function CheckoutTopUpToggle({
+	checked,
+	onChange,
+	contributionType,
+	threshold = 10,
```

Comment:
Not sure about defaulting to 10 here, as this is only appropriate for monthly not annual? I'm wondering why we need to define a default for threshold? In what scenario will it be undefined? 
I'd expect we'd only be rendering this component if threshold is defined further up the Component tree.

Reply:
Well spotted, I think that was just added to get it working quickly in storybook. Have removed and renamed the prop to be more descriptive